apps/backend/fixtures/lambda/reconcile.json

```diff
@@ -1,6 +1,6 @@
 {
   "fiscal_start_date": "2023-01-01",
-  "fiscal_close_date": "2023-02-27",
+  "fiscal_close_date": "2023-03-02",
```

Comment:
This range is just for testing purposes, right? What's the best way to mark it so that it gets changed to the real fiscal year before this is being used in production?

Reply:
I think we could remove the dates from the event and just use an environment variable to set the date. If the plan is to run reconciliation daily starting from the start of the fiscal year, or quarter, etc, then we shouldn't need to pass these in. 
We also probably don't need the option to pass in locations or programs either. It makes sense to look up by location for reporting, but if this is meant to run daily for all locations and programs then we can set these with env vars... I think we weren't clear on the criteria so it was left as flexible as possible for the time being.

Reply:
(I think this is something we should clarify with the team though and could be handled in another PR as this one will unblock testing).